client-java/instrumentation/src/test/java/com/foo/somedifferentpackage/examples/methodreplacement/RedisCrudOperationsImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.foo.somedifferentpackage.examples.methodreplacement;
+
```

Comment:
move these classes to com.foo.somedifferentpackage.examples.methodreplacement.redis